jetty-server/src/main/java/org/eclipse/jetty/server/HttpConnection.java

```diff
@@ -827,7 +827,7 @@ public Action process() throws Exception
                     case DONE:
                     {
                         // If shutdown after commit, we can still close here.
```

Comment:
I'm not sure this comment is correct, or describes enough what is occurring here?